common/lib/xmodule/xmodule/modulestore/inheritance.py

```diff
@@ -98,7 +98,12 @@ class InheritanceMixin(XBlockMixin):
     )
     rerandomize = String(
         display_name=_("Randomization"),
-        help=_("Specify how often variable values in a problem are randomized when a student loads the problem. Valid values are \"always\", \"onreset\", \"never\", and \"per_student\". This setting only applies to problems that have randomly generated numeric values."),
+        help=_(
+            'Specify the default for how often variable values in a problem are randomized. '
+            'This setting should be set to \"never\" unless you plan to provide a Python '
+            'script to identify and randomize values in most of the problems in your course. '
```

Comment:
I'm not finding the revision more clear than the previous version. "...unless you plan to ...randomize values in *most* of the problems in your course" seems vague and raises questions.
 More details about the impact of setting the default to "never" and for the other values?